examples/structured_2d_dgsem/elixir_shallowwater_parabolic_bowl.jl

```diff
@@ -0,0 +1,112 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations2D(gravity_constant=9.81)
+
+"""
+    initial_condition_parabolic_bowl(x, t, equations:: ShallowWaterEquations2D)
+
+Well-known initial condition to test the [`hydrostatic_reconstruction_chen_noelle`](@ref) and its
+wet-dry mechanics. This test has analytical solutions. The initial condition is defined by the 
+analytical solution at time t=0. The bottom topography defines a bowl and the water level is given
+by an oscillating lake. The simulation may take a couple of minute due to the higher resolution.
```

Comment:
```suggestion
by an oscillating lake.
```
We typically do not mention simulation times in docstrings.